src/patterns/national-insurance-numbers/index.md.njk

```diff
@@ -35,7 +35,7 @@ Show a National Insurance number using the format ‘QQ 12 34 56 C’ — the sp
 When asking for a National Insurance number:
 
 - allow for 13 characters as National Insurance numbers are spaced in pairs followed by a single letter
-- allow upper and lower case letters and strip out spaces before validating
+- let users enter upper and lower case letters, additional spaces and punctuation, and remove any unwanted characters before validating
```

Comment:
Have seen the words 'remove' and 'ignore' used interchangeably throughout this content. Is there a difference, or should we just use whichever of the 2 words is more appropriate?

Reply:
Asked the team about this – the summary really is that 'remove' implies you manipulate original value, where 'ignore' doesn't. 

For example, given the NI number ‘QQ 12 34 56 C’:
* If spaces are removed, the string would be validated as ‘QQ123456C’ and on later recall (such as a review step) would show the number as ‘QQ123456C’
* If spaces are ignored, the string would be validated as ‘QQ123456C’ but a later review step would display it ‘QQ 12 34 56 C’, as it was entered

Which one you use will depend a lot on the kind of data being handled, how it's stored, and how you want to present it back to users. But we reckon for most teams in most circumstances, 'ignore' covers it.

Reply:
Thanks for investigating!

Based on that, maybe there's a case for mentioning both words, but with 'ignore' coming first.

Let's check it with @calvin-lau-sig7 when he's back.